federation/pkg/kubefed/init/init.go

```diff
@@ -62,6 +63,16 @@ const (
 	AdminCN                     = "admin"
 	HostClusterLocalDNSZoneName = "cluster.local."
 
+	// User name used by federation controller manager to make
```

Comment:
Dont we need similar constant for federation controller manager to talk to host cluster?